src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientGameoverSystem.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.ligthandshadow.componentsystem.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.las.UI.GameoverScreen;
+import org.terasology.ligthandshadow.componentsystem.components.LASTeamComponent;
+import org.terasology.ligthandshadow.componentsystem.events.GameoverEvent;
+import org.terasology.logic.players.LocalPlayer;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.NUIManager;
+
+@RegisterSystem(RegisterMode.CLIENT)
+public class ClientGameoverSystem extends BaseComponentSystem {
+    private static final Logger logger = LoggerFactory.getLogger(ClientGameoverSystem.class);
+    @In
```

Comment:
Very minor and the code style rules don't cover it but we generally newline apart different blocks of variables - so a blank line between `private static ...` and `private ...` variables. Sometimes between vars inside each group, but that's more up to whatever feels right :-)